.cirrus.yml

```diff
@@ -10,7 +10,7 @@ env:
 
 task:
   name: Arm64 Graviton2 / $CC
-  skip: "changesIncludeOnly('doc/**', '**.{md,rdoc}', '.document')"
+  skip: "changesIncludeOnly('doc/**', '**.{md,rdoc}')"
```

Comment:
This one-line change seems like it probably shouldn't be in this PR

Reply:
:+1: removed that from https://github.com/ruby/ruby/pull/6289/commits/3f96272dda3071547a6f13291afa5ab8b89d7d75